packages/cashier/src/assets/svgs/trading-platform/svg.d.ts

```diff
@@ -0,0 +1,4 @@
+declare module '*.svg' {
+    const content: React.SVGAttributes<SVGElement>;
+    export default content;
+}
```

Comment:
@hamid-deriv We already have this in `global.d.ts`, We just need to update the file like [this](https://github.com/binary-com/deriv-app/pull/7918/files#diff-581da69f6fb74fc5bad7d881e1d39e33bcf04d7c064216fd84ef859a6f86fc78) to fix it.

Reply:
Sorted.

Reply:
Fixed.